core/translate/Translate.cpp

```
@@ -92,11 +126,11 @@ namespace atlas
         }
     }
 
-    template <typename XLEN, MMUMode MODE, bool INST_TRANSLATION>
+    template <typename XLEN, MMUMode MODE, bool TRANSLATION>
```

You don't need to include this in your PR, but we will need to change this bool to some new enum for the access type (inst, load or store) so we can throw the appropriate exception type.

---

Already thought of that.  :grin:   I actually started implementing that so I could remove the `is_store_type_` in AtlasInst.hpp, but decided to go ahead with this PR